bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/DahuaHandler.java

```diff
@@ -273,7 +356,6 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     // If a camera does not need to poll a request as often as snapshots, it can be
     // added here. Binding steps through the list.
     public ArrayList<String> getLowPriorityRequests() {
```

Comment:
Return types should be as general as possbile.
```suggestion
    public List<String> getLowPriorityRequests() {
```